components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```
@@ -153,27 +158,26 @@ public static String getUserInfoClaimDialect() {
      * @param appName
      * @return
      */
-    public static String getErrorPageURL(String errorCode, String errorMessage, String appName, String redirect_uri) {
+    public static String getErrorPageURL(String errorCode, String errorMessage, String appName, String redirectUri) {
 
-        String errorPageUrl = null;
-        if (redirect_uri != null && !redirect_uri.equals("")) {
-            errorPageUrl = redirect_uri;
+        String errorPageUrl;
+        if (redirectUri != null && !"".equals(redirectUri)) {
+            errorPageUrl = redirectUri;
         } else {
             errorPageUrl = CarbonUIUtil.getAdminConsoleURL("/") + "../authenticationendpoint/oauth2_error.do";
         }
         try {
-            errorPageUrl += "?" + OAuthConstants.OAUTH_ERROR_CODE + "="
-                    + URLEncoder.encode(errorCode, "UTF-8") + "&" + OAuthConstants.OAUTH_ERROR_MESSAGE + "="
-                    + URLEncoder.encode(errorMessage, "UTF-8");
+            errorPageUrl += "?" + OAuthConstants.OAUTH_ERROR_CODE + "=" + URLEncoder.encode(errorCode, "UTF-8") + "&"
+                    + OAuthConstants.OAUTH_ERROR_MESSAGE + "=" + URLEncoder.encode(errorMessage, "UTF-8");
         } catch (UnsupportedEncodingException e) {
-            // ignore
+            log.error("Error while encoding the error page url", e);
```

log ignored exceptions in debug level